CMakeLists.txt

```diff
@@ -55,42 +55,42 @@ set(GEOSChem_DETECTED_FORTRAN_COMPILER_VERSION ${CMAKE_Fortran_COMPILER_VERSION}
 )
 
 set(GEOSChem_Fortran_FLAGS_Intel
-    -cpp -w -auto -noalign "SHELL:-convert big_endian" "SHELL:-fp-model source"
+    -g -cpp -w -auto -noalign "SHELL:-convert big_endian" "SHELL:-fp-model source"
```

Comment:
Why is debugging information now turned on by default? Shouldn't it only be if building in DEBUG mode? Same comment for GNU.

Reply:
So the -g flag generates debugging information that is used when you call up GDB debugger.  

Say the run stops with an error.  You can use the command `gdb gcclassic core.xxxxx` to go right to the point of the error.  Without the -g flag you will see a bunch of gibberish in hex.  With -g, that is translated into source code line numbers.  

I added this so that the GCST could more quickly find out where user's runs stopped.  Before this, the -g was only applied when you configured with -DCMAKE_BUILD_TYPE=Debug.  But most users who encounter errors are using Release instead of Debug.  So when we get a Github issue with an unknown stop, we can tell users to run gdb and find the line where it stopped.

Reply:
Does including the -g flag for regular runs have any performance hits?

Reply:
Ah, I see Melissa asked this question later in this conversation.